src/core/include/openvino/core/preprocess/input_tensor_info.hpp

```diff
@@ -118,6 +119,17 @@ class OPENVINO_API InputTensorInfo final {
     ///
     /// \return Reference to 'this' to allow chaining with other calls in a builder-like manner.
     InputTensorInfo& set_shape(const ov::PartialShape& shape);
+
+    /// \brief Helper function to reuse element type and shape from user's created tensor. Use this only in case if
+    /// input tensor is already known and available before. Overwrites previously set element type & shape via
+    /// `set_element_type` and `set_shape`. Tensor's memory type is not reused, so if `runtime_tensor` represents remote
+    /// tensor with particular memory type - you should still specify appropriate memory type manually using
+    /// `set_memory_type`
+    ///
+    /// \param runtime_tensor User's created tensor.
+    ///
+    /// \return Reference to 'this' to allow chaining with other calls in a builder-like manner.
+    InputTensorInfo& from(const ov::runtime::Tensor& runtime_tensor);
```

Comment:
can we demonstrate its usage in current samples? Some of them create tensor first, next try to apply preprocessing.

Reply:
Done, updated python's hello_classification. I can also update C++ hello_classification, if needed.

Reply:
>  I can also update C++ hello_classification, if needed.

Yes, let's use new API whether it's appropriate. At least here https://github.com/openvinotoolkit/openvino/blob/master/samples/cpp/hello_classification/main.cpp#L67-L86

BTW, if I use `set_from`, but use `set_layout()` and `set_spatial_dynamic_shape()` then, does the order matters and custom layout / shapes will be used? Let's clarify this in docs

Reply:
please, address

Reply:
Done. C++ hello_classification is updated. Added note to description of set_from that it should not be used together with 'set_spatial...` methods, same as current behavior for `set_shape`